Mathlib/CategoryTheory/Bicategory/Opposites.lean

```diff
@@ -0,0 +1,230 @@
+/-
+Copyright (c) 2025 Calle Sönne. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Calle Sönne
+-/
+
+import Mathlib.CategoryTheory.Bicategory.Basic
+import Mathlib.CategoryTheory.Opposites
+
+/-!
+# Opposite bicategories
+
+We construct the 1-cell opposite of a bicategory `B`, called `Bᵒᵖ`. It is defined as follows
+* The objects of `Bᵒᵖ` correspond to objects of `B`.
+* The morphisms `X ⟶ Y` in `Bᵒᵖ` are the morphisms `Y ⟶ X` in `B`.
+* The 2-morphisms `f ⟶ g` in `Bᵒᵖ` are the 2-morphisms `f ⟶ g` in `B`. In other words, the
+  directions of the 2-morphisms are preserved.
+
+Note that the standard notation for the opposite of a bicategory is `Bᵒᵖ`, however this clashes
+with the notation for the opposite of a 1-category, so we use `Bᵒᵖ` instead.
```

Comment:
(1) Those two notations look identical to me. (2) Where do you setup this notation? I don't see it in the file.